### ISSUE TITLE
runtime: timeout with GDB tests on aix/ppc64

### ISSUE THREAD

**Issue by Helflym** [golang/go#35710]
https://build.golang.org/log/98c7edecebf8de8d333d58a2286729cd6d821cef
https://build.golang.org/log/66fbb96c7aee8d5b6f191319ed3d8748f238255f
https://build.golang.org/log/8fba4a8ce0f6fb1cd3170013f51ff2fdbc35d269

Our GDB on AIX is a bit unstable and it doesn't seem to like the new page allocator for an unknown reason. Even an helloworld can't be run properly. It might be related to #35451 but I think it's more a GDB problem. The team working on it has been contacted. So until now, I would like to disable these tests in order to release our builder.

**Comment by mknyszek** [comment]
@Helflym can you be more specific about how it doesn't like the new page allocator? Do you know if https://go-review.googlesource.com/c/go/+/207497/5 helps? (It should reduce the size of the PROT_NONE mappings from 9ish GiB to about 600 MiB).

**Comment by mknyszek** [comment]
I'm sorry, what I really meant to ask was, do you know for sure that it's the new page allocator it doesn't like? There were a number of other big changes that went into this release (e.g. asynchronous preemption). Maybe it really dislikes getting a lot of signals? I could see that being some very slow path in a gdb port.

**Comment by Helflym** [comment]
@mknyszek no, I'm sorry I thought it was related to the new page allocator. But indeed, as you said, it seems to be linked with 374c2847f9c03da7365bfb78e5ef96a0cb837656 according to a `git bisect`. Moreover, our GDB already have trouble handling signals, so it seems legit. 
I'm waiting for updates from the GDB team anyway.
